src/Tmds.Ssh/SftpClient.File.cs

```diff
@@ -0,0 +1,160 @@
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Buffers;
+using System.Buffers.Binary;
+
+namespace Tmds.Ssh
+{
+    public sealed class SftpFile
+    {
+        private readonly byte[] _handle;
+
+        internal SftpFile(byte[] handle)
+        {
+            _handle = handle;
+        }
+
+        // For now I am taking a client as an argument because not sure how to do this
+        public ValueTask<bool> CloseAsync(SftpClient client) => client.CloseFileAsync(_handle);
```

Comment:
To get rid of the argument here, you need to pass `SftpClient` to `OpenFileOperation` as an argument, which can then pass it to `SftpFile` constructor.

Reply:
Yes, I forgot to push the changes, done